src/Orchard.Web/Modules/Orchard.Users/Module.cs

```diff
@@ -29,11 +30,12 @@ public Startup(ShellSettings shellSettings, IOptions<IdentityOptions> options)
 
         public override void Configure(IApplicationBuilder builder, IRouteBuilder routes, IServiceProvider serviceProvider)
         {
+            builder.UseIdentity();
```

Comment:
Either use `app.UseIdentity()` or add the cookies middleware manually but not both at the same time. You'll get duplicate middleware instances.